src/app/leaves/actions/action.test.ts

```diff
@@ -0,0 +1,24 @@
+import { waitFor } from "@testing-library/dom";
```

Comment:
File name is wrong. It should depict the action name.

Reply:
Fixed this one